lib/User/Manager.php

```diff
@@ -241,7 +241,7 @@ public function getCachedEntry($uid) {
 		}
 
 		// should have been cached during login or while iterating over multiple users, eg during sync
-		$this->logger->warning('No cached ldap result found for '. $uid , ['app' => self::class]);
+		$this->logger->info('No cached ldap result found for '. $uid , ['app' => self::class]);
```

Comment:
Agree that `warning` is a bit to harsh in that case? - but why `info` and not `debug` ? cc @butonic